<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
  <meta charset="utf-8" />
  <meta name="generator" content="pandoc" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
  <title>resume</title>
  <style>
      code{white-space: pre-wrap;}
      span.smallcaps{font-variant: small-caps;}
      span.underline{text-decoration: underline;}
      div.column{display: inline-block; vertical-align: top; width: 50%;}
  </style>
  <!--[if lt IE 9]>
    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv-printshiv.min.js"></script>
  <![endif]-->
  <style type="text/css">
  /*
   * Copyright 2013 Christophe-Marie Duquesne <chmd@chmd.fr>
   *
   * CSS for making a resume with pandoc. Inspired by moderncv.
   *
   * This CSS document is delivered to you under the CC BY-SA 3.0 License.
   * https://creativecommons.org/licenses/by-sa/3.0/deed.en_US
   */
  
  /* Whole document */
  body {
      font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
      max-width: 800px;
      margin: auto;
      background: #FFFFFF;
      padding: 10px 10px 10px 10px;
  }
  
  /* Title of the resume */
  h1 {
      font-size: 55px;
      color: #757575;
      text-align:center;
      margin-bottom:15px;
  }
  /*h1:hover {
      background-color: #757575;
      color: #FFFFFF;
      text-shadow: 1px 1px 1px #333;
  }*/
  
  /* Titles of categories */
  h2 {
      /* This is called "sectioncolor" in the ConTeXt stylesheet. */
      color: #397249;
  }
  /* There is a bar just before each category */
  h2:before {
      content: "";
      display: inline-block;
      margin-right:1%;
      width: 16%;
      height: 10px;
      /* This is called "rulecolor" in the ConTeXt stylesheet. */
      background-color: #9CB770;
  }
  /*h2:hover {
      background-color: #397249;
      color: #FFFFFF;
      text-shadow: 1px 1px 1px #333;
  }*/
  
  /* Definitions */
  dt {
      float: left;
      clear: left;
      width: 17%;
      /*font-weight: bold;*/
  }
  dd {
      margin-left: 17%;
  }
  p {
      margin-top:0;
      margin-bottom:7px;
  }
  
  /* Blockquotes */
  blockquote {
      text-align: center
  }
  
  /* Links */
  a {
      text-decoration: none;
      color: #397249;
  }
  a:hover, a:active {
      background-color: #397249;
      color: #FFFFFF;
      text-decoration: none;
      text-shadow: 1px 1px 1px #333;
  }
  
  /* Horizontal separators */
  hr {
      color: #21618C
      /*#A6A6A6;*/
  }
  
  table {
      width: 100%;
  }
  </style>
</head>
<body>
<h1 id="theodore-natoli">Theodore Natoli</h1>
<!-- contact info center justified -->
<blockquote>
<p><a href="mailto:ted.e.natoli@gmail.com">ted.e.natoli@gmail.com</a> • (857)-498-1946 • 109 Green St. Wakefield, MA 01880</p>
</blockquote>
<blockquote>
<p><a href="https://github.com/tnat1031">Github</a> • <a href="https://www.researchgate.net/profile/Ted_Natoli2">Research Gate</a> • <a href="https://www.linkedin.com/in/ted-natoli-compbio">LinkedIn</a></p>
</blockquote>
<!-- quote or summary statement set off by horizontal lines -->
<hr />
<blockquote>
<p>I am an experienced data scientist seeking an opportunity to advance biomedical research.</p>
</blockquote>
<hr />
<h2 id="professional-experience">Professional Experience</h2>
<dl>
<dt>2012-present</dt>
<dd><p><strong>Computational Biologist</strong> • <em>The Broad Institute in Cambridge, MA</em></p>
<p><em>Previously:</em> Associate Computational Biologist, Research Associate</p>
<p>As a member of the Connectivity Map (CMap) group within the Broad’s Cancer Program, I serve as a lead data analyst and mentor to junior analysts on projects utilizing data from the L1000 high-throughput gene expression profiling assay. Responsibilities and achievements include:</p>
<ul>
<li><p>Lead analysis strategy on internal and collaborative projects with academia and pharma in areas such as drug repurposing, drug discovery, and hit characterization.</p></li>
<li><p>Effectively visualize complex data and present analysis methods and results to scientific audiences in various domains.</p></li>
<li><p>Develop analysis methods for integrating L1000 data with other high-throughput readouts such as cell viability and cell morphology.</p></li>
<li><p>Lead efforts to structure and run crowdsourcing competitions which have resulted in improvements to 3 core CMap algorithms.</p></li>
<li><p>Advance CMap software development projects by acting as computational liaison for software engineers.</p></li>
<li><p>Develop and maintain library of CMap R code.</p></li>
<li><p>Accelerate L1000 assay development by assisting the data generation team with ad-hoc analyses.</p></li>
<li><p>Improved efficiency and reliability of the L1000 data processing pipeline by writing utilities in Python and utilizing automated scheduling programs such as cron and Jenkins.</p></li>
<li><p>Improved L1000 data tracking and organization by implementing a LIMS database and web-based user interface to assist with managing the data production queue, tracking data generation batches, and easily viewing QC reports.</p></li>
<li><p>Lead technical support and outreach efforts within the CMap user community.</p></li>
</ul>
</dd>
</dl>
<!-- split out RA position at Broad? -->
<dl>
<dt>2009-2012</dt>
<dd><p><strong>Bioinformatics Scientist</strong> • <em>Ligon Discovery in Cambridge, MA</em></p>
<p><em>Previously:</em> Research Associate</p>
<p>Led bioinformatics efforts for drug discovery in a high-throughput screening (HTS) setting by processing, managing, analyzing, and presenting data from small molecule microarray (SMM) and other biological assays. Specifically:</p>
<ul>
<li><p>Increased SMM data analysis efficiency by building and implementing a pipeline using Perl and R.</p></li>
<li><p>Designed, built, and administered a MySQL LIMS database.</p></li>
<li><p>Implemented and ran SMM hit-calling analyses using R.</p></li>
<li><p>Managed and executed SMM projects for clients according to specified timelines and presented corresponding analysis results.</p></li>
</ul>
</dd>
<dt>2007-2009</dt>
<dd><p><strong>Biologist and Techincal Support Engineer</strong> • <em>Hamilton Thorne, Inc. in Beverly, MA</em></p>
<ul>
<li><p>Independently conducted biological R&amp;D experiments.</p></li>
<li><p>Trained repair technicians and customers on proprietary Computer Assisted Semen Analysis (CASA) hardware and software and the applications thereof.</p></li>
<li><p>Performed mechanical, electrical and software validation, maintenance, and testing of CASA systems.</p></li>
</ul>
</dd>
<dt>2007</dt>
<dd><p><strong>Imaging Assistant</strong> • <em>Perceptive Informatics (PAREXEL) in Waltham, MA</em></p>
<ul>
<li><p>Developed and coordinated clinical study-related activities in compliance with FDA regulatory requirements.</p></li>
<li><p>Used computer systems to convert medical image data to digital form.</p></li>
<li><p>Maintained a database of medical images and associated results.</p></li>
</ul>
</dd>
</dl>
<h2 id="education">Education</h2>
<dl>
<dt>2014</dt>
<dd><p><strong>ALM, Biotechnology with Concentration in Bioinformatics</strong>; Harvard Extension School (Cambridge, MA)</p>
<p><em>Thesis:</em> Implemented a full-stack web application for visualizing and analyzing CMap query results as a network graph.</p>
</dd>
<dt>2007</dt>
<dd><strong>BS, Biomedical Engineering</strong>; Boston University (Boston, MA)
</dd>
</dl>
<h2 id="technical-experience">Technical Experience</h2>
<dl>
<dt>Analysis</dt>
<dd>Clustering, regression, dimensionality reduction, feature selection, visualization
</dd>
<dt>Programming Languages</dt>
<dd><strong>R:</strong> Over 8 years of experience using R for exploratory data analysis, data visualization, and analysis workflows. Proficient in many packages including <a href="https://cran.r-project.org/web/packages/data.table/vignettes/datatable-intro.html">data.table</a>, <a href="https://ggplot2.tidyverse.org/">ggplot2</a>, <a href="https://shiny.rstudio.com/">shiny</a>, and others. Author of the <a href="https://github.com/cmap/cmapR">cmapR</a> package for manipulating annotated gene expression matrices.
</dd>
<dd><p><strong>Python:</strong> Over 4 years of experience using Python for data wrangling and writing scripts for web scraping and data processing pipeline automtation. Familiar with the <a href="https://pandas.pydata.org/">pandas</a> and <a href="https://www.numpy.org/">numpy</a> packages for data analysis.</p>
</dd>
<dd><p>Basic knowledge of <strong>SQL</strong>, <strong>Matlab</strong>, <strong>Javascript</strong> including <strong>Node.js</strong>, <strong>HTML/CSS</strong>, <strong>Markdown</strong>, <strong>LaTex</strong></p>
</dd>
<dt>Databases</dt>
<dd><strong>MySQL</strong>, <strong>MongoDB</strong>
</dd>
<dt>Cloud</dt>
<dd><strong>AWS</strong> EC2 and S3, <strong>Google</strong> BigQuery
</dd>
<dt>Other</dt>
<dd><strong>Git</strong>, <strong>Docker</strong>, <strong>Tableau</strong>
</dd>
</dl>
<h2 id="selected-publications">Selected Publications</h2>
<ul>
<li><p>Subramanian, Aravind, Rajiv Narayan, Steven M. Corsello, David D. Peck, <strong>Ted E. Natoli,</strong> Xiaodong Lu, Joshua Gould, et al. 2017. <a href="https://www.ncbi.nlm.nih.gov/pubmed/29195078">“A Next Generation Connectivity Map: L1000 Platform and the First 1,000,000 Profiles.”</a> Cell 171 (6): 1437–52.e17.</p></li>
<li><p>Amy Li, Xiaodong Lu, <strong>Ted Natoli,</strong> Joshua Bittker, Nisha S. Sipes, Aravind Subramanian, Scott Auerbach, David H. Sherr, and Stefano Monti. 2019. <a href="https://www.ncbi.nlm.nih.gov/pubmed/30964323">“The Carcinogenome Project: In Vitro Gene Expression Profiling of Chemical Perturbations to Predict Long-Term Carcinogenicity.”</a> Environmental Health Perspectives 127 (4).</p></li>
<li><p>Smith, Ian, Peyton G. Greenside, <strong>Ted Natoli,</strong> David L. Lahr, David Wadden, Itay Tirosh, Rajiv Narayan, et al. 2017. <a href="https://www.ncbi.nlm.nih.gov/pubmed/29190685">“Evaluation of RNAi and CRISPR Technologies by Large-Scale Gene Expression Profiling in the Connectivity Map.”</a> PLoS Biology 15 (11): e2003213.</p></li>
<li><p>Berger, Alice H., Angela N. Brooks, Xiaoyun Wu, Yashaswi Shrestha, Candace Chouinard, Federica Piccioni, Mukta Bagul, et al. 2016. <a href="https://www.ncbi.nlm.nih.gov/pubmed/27478040">“High-Throughput Phenotyping of Lung Cancer Somatic Mutations.”</a> Cancer Cell 30 (2): 214–28.</p></li>
</ul>
<hr />
</body>
</html>
