<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
  <meta charset="utf-8" />
  <meta name="generator" content="pandoc" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
  <title>resume</title>
  <style>
    html {
      line-height: 1.7;
      font-family: Georgia, serif;
      font-size: 20px;
      color: #1a1a1a;
      background-color: #fdfdfd;
    }
    body {
      margin: 0 auto;
      max-width: 40em;
      padding-left: 50px;
      padding-right: 50px;
      padding-top: 50px;
      padding-bottom: 50px;
      hyphens: auto;
      word-wrap: break-word;
      text-rendering: optimizeLegibility;
      font-kerning: normal;
    }
    @media (max-width: 600px) {
      body {
        font-size: 0.9em;
        padding: 1em;
      }
    }
    @media print {
      body {
        background-color: transparent;
        color: black;
      }
      p, h2, h3 {
        orphans: 3;
        widows: 3;
      }
      h2, h3, h4 {
        page-break-after: avoid;
      }
    }
    p {
      margin-top: 1.7em;
    }
    a {
      color: #1a1a1a;
    }
    a:visited {
      color: #1a1a1a;
    }
    img {
      max-width: 100%;
    }
    h1, h2, h3, h4, h5, h6 {
      margin-top: 1.7em;
    }
    ol, ul {
      padding-left: 1.7em;
      margin-top: 1.7em;
    }
    li > ol, li > ul {
      margin-top: 0;
    }
    blockquote {
      margin: 1.7em 0 1.7em 1.7em;
      padding-left: 1em;
      border-left: 2px solid #e6e6e6;
      font-style: italic;
    }
    code {
      font-family: Menlo, Monaco, 'Lucida Console', Consolas, monospace;
      background-color: #f0f0f0;
      font-size: 85%;
      margin: 0;
      padding: .2em .4em;
    }
    pre {
      line-height: 1.5em;
      padding: 1em;
      background-color: #f0f0f0;
      overflow: auto;
    }
    pre code {
      padding: 0;
      overflow: visible;
    }
    hr {
      background-color: #1a1a1a;
      border: none;
      height: 1px;
      margin-top: 1.7em;
    }
    table {
      border-collapse: collapse;
      width: 100%;
      overflow-x: auto;
      display: block;
    }
    th, td {
      border-bottom: 1px solid lightgray;
      padding: 1em 3em 1em 0;
    }
    header {
      margin-bottom: 6em;
      text-align: center;
    }
    nav a:not(:hover) {
      text-decoration: none;
    }
    code{white-space: pre-wrap;}
    span.smallcaps{font-variant: small-caps;}
    span.underline{text-decoration: underline;}
    div.column{display: inline-block; vertical-align: top; width: 50%;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
    ul.task-list{list-style: none;}
    .display.math{display: block; text-align: center; margin: 0.5rem auto;}
  </style>
  <!--[if lt IE 9]>
    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv-printshiv.min.js"></script>
  <![endif]-->
  <style type="text/css">
  /*
   * Copyright 2013 Christophe-Marie Duquesne <chmd@chmd.fr>
   *
   * CSS for making a resume with pandoc. Inspired by moderncv.
   *
   * This CSS document is delivered to you under the CC BY-SA 3.0 License.
   * https://creativecommons.org/licenses/by-sa/3.0/deed.en_US
   */

  /* Whole document */
  body {
      font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
      max-width: 800px;
      margin: auto;
      background: #FFFFFF;
      padding: 10px 10px 10px 10px;
  }

  /* Title of the resume */
  h1 {
      font-size: 55px;
      color: #757575;
      text-align:center;
      margin-bottom:15px;
  }
  /*h1:hover {
      background-color: #757575;
      color: #FFFFFF;
      text-shadow: 1px 1px 1px #333;
  }*/

  /* Titles of categories */
  h2 {
      /* This is called "sectioncolor" in the ConTeXt stylesheet. */
      color: #397249;
  }
  /* There is a bar just before each category */
  h2:before {
      content: "";
      display: inline-block;
      margin-right:1%;
      width: 16%;
      height: 10px;
      /* This is called "rulecolor" in the ConTeXt stylesheet. */
      background-color: #9CB770;
  }
  /*h2:hover {
      background-color: #397249;
      color: #FFFFFF;
      text-shadow: 1px 1px 1px #333;
  }*/

  /* Definitions */
  dt {
      float: left;
      clear: left;
      width: 17%;
      /*font-weight: bold;*/
  }
  dd {
      margin-left: 17%;
  }
  p {
      margin-top:0;
      margin-bottom:7px;
  }

  /* Blockquotes */
  blockquote {
      text-align: center
  }

  /* Links */
  a {
      text-decoration: none;
      color: #397249;
  }
  a:hover, a:active {
      background-color: #397249;
      color: #FFFFFF;
      text-decoration: none;
      text-shadow: 1px 1px 1px #333;
  }

  /* Horizontal separators */
  hr {
      color: #21618C
      /*#A6A6A6;*/
  }

  table {
      width: 100%;
  }
  </style>
</head>
<body>
<h1 id="theodore-natoli">Theodore Natoli</h1>
<!-- contact info center justified -->
<blockquote>
<p><a href="mailto:ted.e.natoli@gmail.com">ted.e.natoli@gmail.com</a> • (857)-498-1946 • Wakefield, MA 01880</p>
</blockquote>
<blockquote>
<p><a href="https://github.com/tnat1031">Github</a> • <a href="https://www.researchgate.net/profile/Ted_Natoli2">Research Gate</a> • <a href="https://www.linkedin.com/in/ted-natoli-compbio">LinkedIn</a></p>
</blockquote>
<!-- quote or summary statement set off by horizontal lines -->
<hr />
<blockquote>
<p>I am an experienced computational biologist seeking an opportunity to advance biomedical research.</p>
</blockquote>
<hr />
<h2 id="professional-experience">Professional Experience</h2>
<dl>
<dt>2012-present</dt>
<dd><p><strong>Computational Biologist</strong> • <em>The Broad Institute in Cambridge, MA</em></p>
<p><em>Previously:</em> Associate Computational Biologist, Research Associate</p>
<p>As a member of the Connectivity Map (<a href="http://clue.io">CMap</a>) group within the Broad’s Cancer Program, I serve as a lead data analyst and supervisor/mentor to junior analysts (1-2 direct reports) on projects utilizing data from the <a href="https://clue.io/connectopedia/what_is_l1000">L1000</a> high-throughput gene expression profiling assay. Responsibilities and achievements include:</p>
<ul>
<li><p>Lead analysis strategy and execution and aid in experimental design for internal and collaborative projects with academia and pharma in areas such as drug repurposing, drug discovery, and hit characterization.</p></li>
<li><p>Effectively visualize complex data and present analysis methods and results to scientific audiences in various domains.</p></li>
<li><p>Develop analysis methods for integrating L1000 data with other high-throughput readouts such as cell viability and cell morphology.</p></li>
<li><p>Lead efforts to structure and run crowdsourcing competitions which have resulted in improvements to 3 core CMap algorithms.</p></li>
<li><p>Advance CMap software development projects by acting as computational liaison for software engineers.</p></li>
<li><p>Develop and maintain a library of CMap R code.</p></li>
<li><p>Accelerate L1000 assay development by assisting the data generation team with ad-hoc analyses.</p></li>
<li><p>Improved efficiency and reliability of the L1000 data processing pipeline by writing utilities in Python and leveraging automated scheduling utilities such as cron and Jenkins.</p></li>
<li><p>Improved L1000 data tracking and organization by implementing a LIMS database and web-based user interface to assist with managing the data production queue, tracking data generation batches, and easily viewing QC reports.</p></li>
<li><p>Host office hours and lead technical support and outreach efforts within the CMap user community.</p></li>
</ul>
</dd>
</dl>
<!-- split out RA position at Broad? -->
<dl>
<dt>2009-2012</dt>
<dd><p><strong>Bioinformatics Scientist</strong> • <em>Ligon Discovery in Cambridge, MA</em></p>
<p><em>Previously:</em> Research Associate</p>
<p>Led bioinformatics efforts for drug discovery in a high-throughput screening (HTS) setting by processing, managing, analyzing, and presenting data from small molecule microarray (SMM) and other biological assays. Specifically:</p>
<ul>
<li><p>Increased SMM data analysis efficiency by building and implementing a pipeline using Perl and R.</p></li>
<li><p>Designed, built, and administered a MySQL LIMS database.</p></li>
<li><p>Implemented and ran SMM hit-calling analyses using R.</p></li>
<li><p>Managed and executed SMM projects for clients according to specified timelines and presented corresponding analysis results.</p></li>
</ul>
</dd>
<dt>2007-2009</dt>
<dd><p><strong>Biologist and Techincal Support Engineer</strong> • <em>Hamilton Thorne, Inc. in Beverly, MA</em></p>
<ul>
<li><p>Independently conducted biological R&amp;D experiments.</p></li>
<li><p>Trained repair technicians and customers on proprietary Computer Assisted Semen Analysis (CASA) hardware and software and the applications thereof.</p></li>
<li><p>Performed mechanical, electrical and software validation, maintenance, and testing of CASA systems.</p></li>
</ul>
</dd>
<dt>2007</dt>
<dd><p><strong>Imaging Assistant</strong> • <em>Perceptive Informatics (PAREXEL) in Waltham, MA</em></p>
<ul>
<li><p>Developed and coordinated clinical study-related activities in compliance with FDA regulatory requirements.</p></li>
<li><p>Used computer systems to convert medical image data to digital form.</p></li>
<li><p>Maintained a database of medical images and associated results.</p></li>
</ul>
</dd>
</dl>
<h2 id="consulting">Consulting</h2>
<dl>
<dt>2019</dt>
<dd><p><strong>Diamond Age Data Science</strong> • <em>in Somerville, MA</em></p>
<p>Served as a consultant and advisor for a Diamond Age Data Science client on an analysis project involving public CMap data.</p>
</dd>
<dt>2016</dt>
<dd><p><strong>Navitor Pharmaceuticals</strong> • <em>in Cambridge, MA</em></p>
<p>Performed differential expression analysis on microarray data to generate gene expression signatures of compounds.</p>
</dd>
</dl>
<h2 id="education">Education</h2>
<dl>
<dt>2014</dt>
<dd><p><strong>ALM, Biotechnology with Concentration in Bioinformatics</strong>; Harvard Extension School (Cambridge, MA)</p>
<p><em>Thesis:</em> Implemented a full-stack web application for visualizing and analyzing CMap query results as a network graph.</p>
</dd>
<dt>2007</dt>
<dd><strong>BS, Biomedical Engineering</strong>; Boston University (Boston, MA)
</dd>
</dl>
<h2 id="technical-experience">Technical Experience</h2>
<!-- breaks at end of line add a litte breathing room around sections -->
<dl>
<dt>Data Types</dt>
<dd>Gene expression (<a href="https://clue.io/connectopedia/what_is_l1000">L1000</a>, RNAseq), cell viability (<a href="https://depmap.org/portal/prism/">PRISM</a>, CTG), cell morphology (<a href="https://www.ncbi.nlm.nih.gov/pubmed/27560178">cell painting</a>)<br></br>
</dd>
</dl>
<!-- existing resources I've worked with, might want to update to include links -->
<dl>
<dt>Resources</dt>
<dd>CMap, GEO, CCLE, DepMap, TGCA, GTEx<br></br>
</dd>
<dt>Analysis</dt>
<dd>Clustering, regression, inference, dimensionality reduction, feature selection, visualization<br></br>
</dd>
<dt>OS</dt>
<dd>Linux, Mac, Windows<br></br>
</dd>
<dt>Software Tools</dt>
<dd>GSEA, RSEM, Bowtie, Samtools<br></br>
</dd>
<dt>Programming Languages</dt>
<dd><strong>R:</strong> Over 8 years of experience using R for exploratory data analysis, data visualization, and analysis workflows. Proficient in many packages/resources including <a href="https://www.bioconductor.org/">Bioconductor</a>, <a href="https://cran.r-project.org/web/packages/data.table/vignettes/datatable-intro.html">data.table</a>, <a href="https://ggplot2.tidyverse.org/">ggplot2</a>, <a href="https://shiny.rstudio.com/">shiny</a>, and others. Author of the <a href="http://bioconductor.org/packages/3.11/bioc/html/cmapR.html">cmapR</a> package for manipulating annotated gene expression matrices.<br></br>
</dd>
<dd><p><strong>Python:</strong> Over 4 years of experience using Python for data wrangling and writing scripts for web scraping and data processing pipeline automation. Familiar with the <a href="https://pandas.pydata.org/">pandas</a> and <a href="https://www.numpy.org/">numpy</a> packages for data analysis and the <a href="https://docs.conda.io/en/latest/">conda</a> package management system.<br></br></p>
</dd>
<dd><p>Basic knowledge of <strong>SQL</strong>, <strong>Matlab</strong>, <strong>Javascript</strong> including <strong>Node.js</strong>, <strong>HTML/CSS</strong>, <strong>Markdown</strong>, <strong>LaTex</strong><br></br></p>
</dd>
<dt>Databases</dt>
<dd><strong>MySQL</strong>, <strong>MongoDB</strong><br></br>
</dd>
<dt>Cloud</dt>
<dd><strong>AWS</strong> EC2 and S3, <strong>Google</strong> BigQuery<br></br>
</dd>
<dt>Other</dt>
<dd><strong>Jupyter</strong>, <strong>Git</strong>, <strong>Docker</strong>, <strong>Tableau</strong><br></br>
</dd>
</dl>
<h2 id="selected-publications">Selected Publications</h2>
<ul>
<li><p>Subramanian, Aravind, Rajiv Narayan, Steven M. Corsello, David D. Peck, <strong>Ted E. Natoli,</strong> Xiaodong Lu, Joshua Gould, et al. 2017. <a href="https://www.ncbi.nlm.nih.gov/pubmed/29195078">“A Next Generation Connectivity Map: L1000 Platform and the First 1,000,000 Profiles.”</a> Cell 171 (6): 1437–52.e17.</p></li>
<li><p>Jin, Xin, Zelalem Demere, Karthik Nair, Ahmed Ali, Gino B. Ferraro, <strong>Ted Natoli</strong>, Amy Deik, et al. 2020. <a href="http://dx.doi.org/10.1038/s41586-020-2969-2">“A Metastasis Map of Human Cancer Cell Lines.”</a> Nature 588 (7837): 331–36.</p></li>
<li><p>Amy Li, Xiaodong Lu, <strong>Ted Natoli,</strong> Joshua Bittker, Nisha S. Sipes, Aravind Subramanian, Scott Auerbach, David H. Sherr, and Stefano Monti. 2019. <a href="https://www.ncbi.nlm.nih.gov/pubmed/30964323">“The Carcinogenome Project: In Vitro Gene Expression Profiling of Chemical Perturbations to Predict Long-Term Carcinogenicity.”</a> Environmental Health Perspectives 127 (4).</p></li>
<li><p>Blasco, Andrea, <strong>Ted Natoli,</strong> Michael G. Endres, Rinat A. Sergeev, Steven Randazzo, Jin H. Paik, N. J. Maximilian Macaluso, et al. n.d. <a href="https://doi.org/10.1093/bioinformatics/btab192">“Improving Deconvolution Methods in Biology through Open Innovation Competitions: An Application to the Connectivity Map.”</a> Bioinformatics. April 7, 2021.</p></li>
<li><p>Blasco, Andrea, Michael G. Endres, Rinat A. Sergeev, Anup Jonchhe, Max Macaluso, Rajiv Narayan, <strong>Ted Natoli,</strong> et al. 2019. <a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0222165">“Advancing Computational Biology and Bioinformatics Research Through Open Innovation Competitions.”</a> PLOS ONE 14(9): e0222165.</p></li>
</ul>
<!-- * Litichevskiy, Lev, Ryan Peckner, Jennifer G. Abelin, Jacob K. Asiedu, Amanda L. Creech, John F. Davis, Desiree Davison, et al. 2018. ["A Library of Phosphoproteomic and Chromatin Signatures for Characterizing Cellular Responses to Drug Perturbations."](https://www.ncbi.nlm.nih.gov/pubmed/29655704) Cell Systems 6 (4): 424–43.e7. -->
<ul>
<li>Smith, Ian, Peyton G. Greenside, <strong>Ted Natoli,</strong> David L. Lahr, David Wadden, Itay Tirosh, Rajiv Narayan, et al. 2017. <a href="https://www.ncbi.nlm.nih.gov/pubmed/29190685">“Evaluation of RNAi and CRISPR Technologies by Large-Scale Gene Expression Profiling in the Connectivity Map.”</a> PLoS Biology 15 (11): e2003213.</li>
</ul>
<!-- * Berger, Alice H., Angela N. Brooks, Xiaoyun Wu, Yashaswi Shrestha, Candace Chouinard, Federica Piccioni, Mukta Bagul, et al. 2016. ["High-Throughput Phenotyping of Lung Cancer Somatic Mutations."](https://www.ncbi.nlm.nih.gov/pubmed/27478040) Cancer Cell 30 (2): 214–28. -->
<hr />
</body>
</html>
